src/modules/components/proposal/proposalActions/actions/generators/proposalActionChangeSettingsMultisig.ts

```diff
@@ -0,0 +1,28 @@
+import { type IProposalActionChangeSettings, ProposalActionType } from '../../proposalActionsTypes';
+
+export const generateProposalActionChangeSettingsMultisig = (
```

Comment:
Given that the interface is one `IProposalActionChangeSettings`, I would only implement one generator and keep the change-settings story generic and not plugin-specific